constants/oauth.go

```diff
@@ -3,4 +3,5 @@ package constants
 const (
 	OAuthClientCreateSuccess = "The Client was successfully created"
 	OAuthClientInvalid       = "Authentication client is invalid"
+	TokenInvalid             = "Authenticcation token is invalid"
```

Comment:
Should this one be a code e.g. `invalid_token`?
Because you are using it for the error code, and I think it should be something like above suggestion instead of normal sentence 🤔

Reply:
Ahh yes, I rebased after merged the error improvement branch so this should be fixed 🙇🏼